guides/app-guides/earn/staking.md

```diff
@@ -0,0 +1,37 @@
+---
+description: Earning rewards with Angle
+---
+
+# Earning ANGLE and other rewards
+
+ANGLE tokens are issued every week and distributed to different stakeholders of the Angle ecosystem.
+
+The [Earn page](https://app.angle.money/#/earn) of the app lists all the pools and farms where it is possible to earn ANGLE tokens, and more generally any other yield opportunities linked to Angle.
+
+This is also where you can deposit tokens accepted as collateral in the Core module to act as an [SLP](/core-module/standard-liquidity-providers/README.md).
+
+{% hint style="info" %}
+For more details on how Uniswap V3 Liquidity Providers can be rewarded with ANGLE tokens, check [this guide](/guides/other/univ3-lp.md).
+{% endhint %}
+
+## Deposit, stake, or withdraw tokens as an SLP
+
+In the Earn page, tokens can be deposited to earn yield and over-collateralize agEUR. These funds are invested into DeFi strategies, and can be staked to earn additional ANGLE rewards.
+
+When on the Earn page, click on the SanTokens anchor and select your pool. Then, you can deposit, stake, or withdraw your funds by selecting the token you want to use.
+
+![deposit and stake sanTokens](/.gitbook/assets/sanTokens-earn.png)
+
+## Claiming ANGLE tokens
```

Comment:
Let's use a reference to UniV3 in this guide, or even merge the page with UniV3 incentive system in this since it's all happening here